docs/src/css/custom.css

```
@@ -164,6 +165,68 @@ html[data-theme='dark'] .theme-announcement-bar {
+}
+
+/* Softer border in dark mode */
+html[data-theme='dark'] .navbar {
+  border-bottom-color: rgba(255, 255, 255, 0.08);
+}
+
+/* ---- Homepage frosted-glass navbar ---- */
+html[data-page='home'] .navbar,
+html[data-page='home'] .navbar.navbar--fixed-top {
+  --ifm-navbar-background-color: transparent;
+  background: rgba(255, 255, 255, 0.6) !important;
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)